boundary_layer/schemas/internal/base.py

```diff
@@ -23,7 +23,7 @@
 class JsonSchemaSchema(StrictSchema):
     properties = ma.fields.Dict()
     additionalProperties = ma.fields.Boolean()
-    required = ma.fields.List(ma.fields.String)
+    required = ma.fields.List(ma.fields.String())
```

Comment:
Simply because PyCharm complains with this warning:
```
Expected type 'Field', got 'Type[String]' instead
```

Under the hood, the marshmallow lib does a check and instantiates the class if it's not an already: https://github.com/marshmallow-code/marshmallow/blob/dev/marshmallow/fields.py#L563

Reply:
gotcha, thanks!